src/pages/Login.tsx

```diff
@@ -16,79 +9,47 @@ import {
   yupSchemaLogin,
 } from "../utils/yupSchema";
 import { useFormik } from "formik";
-import { useDispatch, useSelector } from "react-redux";
-import { UserSLice, UsersFromSlice, loginUser } from "../redux/userSlice";
+import { useDispatch } from "react-redux";
+import { loginUser } from "../redux/userSlice";
 import { useAuthContext } from "../context/authContext";
 import { useNavigate } from "react-router-dom";
 import { ROUTES } from "../utils/constans";
 
 export const Login = () => {
-  const [password, setPassword] = useState("");
-  const [email, setEmail] = useState("");
+  const [wrongLogValues, setWrongValues] = useState(true);
   const dispatch = useDispatch();
-  const { loginUser: loginUserAuthContext, isLogin } = useAuthContext();
+  const { loginUser: authLog } = useAuthContext();
   const navigate = useNavigate();
-  const usersDatabase = useSelector((state: UsersFromSlice) => state.user);
 
-  // const formik = useFormik<LoginFormValues>({
-  //   enableReinitialize: true,
-  //   initialValues: initialValuesFormikLogin,
+  const formik = useFormik<LoginFormValues>({
+    enableReinitialize: true,
+    initialValues: initialValuesFormikLogin,
 
-  //   onSubmit: () => {
-  //     console.log("ok");
-  //   },
-  //   validationSchema: yupSchemaLogin,
-  // });
-  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
-    dispatch(loginUser({ password, email }));
-    //  if warunek, zalezy co zwraca dispatch?
-    loginUserAuthContext();
-    navigate(`${ROUTES.home}`);
-  };
+    onSubmit: () => {
+      authLog(formik.values).then((resp) => {
+        if (resp) {
+          navigate(`${ROUTES.home}`);
+          dispatch(loginUser({ email: formik.values.email }));
+          return;
+        } else {
+          setWrongValues(resp);
+        }
```

Comment:
return i tak zatrzymuje egzekucję kodu więc
```suggestion
          return;
        }
          setWrongValues(resp);
        
```